src/editor/serialize.ts

```diff
@@ -42,6 +42,10 @@ export function htmlSerializeIfNeeded(model: EditorModel, {forceHTML = false} =
     if (!parser.isPlainText() || forceHTML) {
         return parser.toHTML();
     }
+    // ensure removal of escape backslashes in non-Markdown messages
+    if (md.indexOf("\\") > -1) {
+        return parser.toPlaintext();
```

Comment:
doesn't this mean we'd lose formatting on stuff like `\*hello\* **world**`? ("hello" should be escaped, but world should not)

Reply:
This is caught by the earlier `toHTML` call (`isPlaintext` returns false for that message), and results in a `formatted_body` of `*hello* <strong>world</strong>` as expected.